crates/sparrow-catalog/catalog/hash.toml

```diff
@@ -50,10 +50,10 @@ time,key,value
 '''
 output_csv = '''
 time,key,value,result
-2021-01-01T00:00:00.000000000,Ben,5,10021492687541564645
-2021-01-01T00:00:00.000000000,Ryan,8,4864632034659211723
-2021-01-02T00:00:00.000000000,Ryan,9,2122274938272070218
-2021-01-03T00:00:00.000000000,Ben,8,4864632034659211723
-2021-01-04T00:00:00.000000000,Ben,,11832085162654999889
-2021-01-04T00:00:00.000000000,Ryan,9,2122274938272070218
+2021-01-01T00:00:00.000000000,Ben,5,16461383214845928621
+2021-01-01T00:00:00.000000000,Ryan,8,6794973171266502674
+2021-01-02T00:00:00.000000000,Ryan,9,15653042715643359010
+2021-01-03T00:00:00.000000000,Ben,8,6794973171266502674
+2021-01-04T00:00:00.000000000,Ben,,0
```

Comment:
Does this imply that `None` of any type hashes to `0`? And is that a different behavior than before (i.e. did `None(Int64)` hash the same as `None(String)`?

Reply:
Previously, they did hash differently. I think this is OK -- we're just saying that our hash function collides on null values of different types. But note we shouldn't be using hash on different types in the same place.